hacks/plashet/build-plashet.py

```diff
@@ -315,7 +315,7 @@ async def cmd_assert_async(cmd: Union[Sequence[str], str], check: bool = True, *
     else:
         cmd_list = cmd
     LOGGER.info("Executing:cmd_assert_async %s", cmd_list)
-    proc = await asyncio.subprocess.create_subprocess_exec(cmd_list[0], *cmd_list[1:], **kwargs)
+    proc = await asyncio.subprocess.create_subprocess_exec(shlex.quote(cmd_list[0]), shlex.quote(cmd_list[1:]), **kwargs)
```

Comment:
using shlex.quote in asyncio.subprocess.create_subprocess_exec seems wrong to me. shlex.quote is used for escaping special characters for shell, however asyncio.subprocess.create_subprocess_exe doesn’t invoke the command in a shell at all.

Reply:
snyk think the first command of asyncio.subprocess.create_subprocess_exe is a shell cmd and has a vulnerability, it not smart to check the env.